charts/px-central/templates/px-backup/pxcentral-backup.yaml

```diff
@@ -22,6 +22,9 @@ metadata:
     app.kubernetes.io/component: px-backup
 {{- include "px-central.labels" . | nindent 4 }}
 rules:
+  - apiGroups: [""]
```

Comment:
Why we require this service access in cluster scope ?

Reply:
we are now vendoring torpedo for auth functionality; and in init() px-backup installed namespace is found by searching all namespaces for the px-backup service. And where the px-backup service is found is marked as px-backup installed ns.
And torpedo requires cluster level service access